src/main/java/com/internet/shop/model/User.java

```diff
@@ -15,11 +16,12 @@ public User(String name, String login, String password) {
         this.password = password;
     }
 
-    public User(Long id, String name, String login, String password, Set<Role> roles) {
+    public User(Long id, String name, String login, String password, byte[] salt, Set<Role> roles) {
         this.id = id;
         this.name = name;
         this.login = login;
         this.password = password;
+        this.salt = salt;
         this.roles = roles;
```

Comment:
![image](https://user-images.githubusercontent.com/50640794/94577131-e0779400-027e-11eb-8616-36b994a4894c.png)
let's modify your constructors like this

Reply:
Ok. Done